Dockerfile

```diff
@@ -0,0 +1,22 @@
+from golang:1.12
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
Not sure if it is my environment problem, but this always fails for me:

Step 1/6 : from golang:1.12
 ---> ffcaee6f7d8b
Step 2/6 : RUN apt-get update && apt-get install --no-install-recommends -y     bc     gcc-multilib     libssl-dev      llvm-dev     libjemalloc-dev     libnuma-dev     python-sphinx     libelf-dev     selinux-policy-dev     libunbound-dev     autoconf     automake     libtool
 ---> Running in 93a1daa2462e
Err:1 http://security.debian.org/debian-security buster/updates InRelease
  Temporary failure resolving 'security.debian.org'
Err:2 http://deb.debian.org/debian buster InRelease
  Temporary failure resolving 'deb.debian.org'
Err:3 http://deb.debian.org/debian buster-updates InRelease
  Temporary failure resolving 'deb.debian.org'
Reading package lists...
W: Failed to fetch http://deb.debian.org/debian/dists/buster/InRelease  Temporary failure resolving 'deb.debian.org'
W: Failed to fetch http://security.debian.org/debian-security/dists/buster/updates/InRelease  Temporary failure resolving 'security.debian.org'
W: Failed to fetch http://deb.debian.org/debian/dists/buster-updates/InRelease  Temporary failure resolving 'deb.debian.org'
W: Some index files failed to download. They have been ignored, or old ones used instead.